includes/admin/reports/class-give-graph.php

```diff
@@ -178,7 +178,7 @@ public function build_graph() {
 
 		$yaxis_count = 1;
 
-		$this->load_scripts();
+//		$this->load_scripts();
```

Comment:
Why is this only commented out? Shouldn't this call and the method be removed completely?

Reply:
@DevinWalker this is not removed because I am waiting for your review on this commit.
Ref: https://github.com/WordImpress/Give/pull/3038#pullrequestreview-112225927

Reply:
That's my comment. It's working, so remove it.

Reply:
@ravinderk please remove this call - it's fine as we're planning on redoing reports and refreshing the screen outputs new graphs in the proper size.

Reply:
@ravinderk I have removed this fn call please review.